### ISSUE TITLE
Log file trace [RC6]

### ISSUE THREAD

**Issue by colin1497** [Tribler/tribler#1767]
After last session closed, got this log file.  I can't tell for sure if there are new issues or not.  Feel free to ignore if this is all duplicate:

ERROR   1448890974.72             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448891263.04             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448892000.47             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448892283.63             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448892770.25             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448895199.19             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1064, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 722, in relay_packet
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1255, in crypto_relay
    
  File "Tribler\community\tunnel\crypto\tunnelcrypto.pyo", line 81, in decrypt_str
    
  File "cryptography\hazmat\primitives\ciphers\base.pyo", line 174, in finalize
    
  File "cryptography\hazmat\backends\openssl\ciphers.pyo", line 137, in finalize
    
cryptography.exceptions.InvalidTag: 

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1064, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 722, in relay_packet
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1255, in crypto_relay
    
  File "Tribler\community\tunnel\crypto\tunnelcrypto.pyo", line 81, in decrypt_str
    
  File "cryptography\hazmat\primitives\ciphers\base.pyo", line 174, in finalize
    
  File "cryptography\hazmat\backends\openssl\ciphers.pyo", line 137, in finalize
    
cryptography.exceptions.InvalidTag: 
ERROR   1448895933.79             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448896344.66             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448896476.31             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448896801.42             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448897054.89             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448897613.01             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448899753.77             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448901513.82             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448902540.84             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448906171.40             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448906978.62             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448908830.01             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448911628.44             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448911684.19             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448912170.63             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448913447.43             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448913951.09             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\dispersy\endpoint.pyo", line 250, in dispersythread_data_came_in
    
  File "Tribler\dispersy\dispersy.pyo", line 1458, in on_incoming_packets
    
  File "Tribler\dispersy\dispersy.pyo", line 1456, in <lambda>
    
exceptions.IndexError: string index out of range

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\dispersy\endpoint.pyo", line 250, in dispersythread_data_came_in
    
  File "Tribler\dispersy\dispersy.pyo", line 1458, in on_incoming_packets
    
  File "Tribler\dispersy\dispersy.pyo", line 1456, in <lambda>
    
exceptions.IndexError: string index out of range
ERROR   1448914102.03             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448914319.37             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448915293.46             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448915576.25             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448917032.44             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448917153.00             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448918707.70             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1064, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 722, in relay_packet
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1255, in crypto_relay
    
  File "Tribler\community\tunnel\crypto\tunnelcrypto.pyo", line 81, in decrypt_str
    
  File "cryptography\hazmat\primitives\ciphers\base.pyo", line 174, in finalize
    
  File "cryptography\hazmat\backends\openssl\ciphers.pyo", line 137, in finalize
    
cryptography.exceptions.InvalidTag: 

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1064, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 722, in relay_packet
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1255, in crypto_relay
    
  File "Tribler\community\tunnel\crypto\tunnelcrypto.pyo", line 81, in decrypt_str
    
  File "cryptography\hazmat\primitives\ciphers\base.pyo", line 174, in finalize
    
  File "cryptography\hazmat\backends\openssl\ciphers.pyo", line 137, in finalize
    
cryptography.exceptions.InvalidTag: 
ERROR   1448919052.96             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448919253.69             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448919432.12             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448919750.92             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448921322.74             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448921966.13             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448924315.82             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448925515.33             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448925848.79             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448925852.23             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448926234.03             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448926304.78             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448928297.42             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448929327.66             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448929343.59             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448929728.82             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1068, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1229, in crypto_in
    
exceptions.TypeError: %d format: a number is required, not tuple
ERROR   1448936626.63             log:443   Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1064, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 722, in relay_packet
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1255, in crypto_relay
    
  File "Tribler\community\tunnel\crypto\tunnelcrypto.pyo", line 81, in decrypt_str
    
  File "cryptography\hazmat\primitives\ciphers\base.pyo", line 174, in finalize
    
  File "cryptography\hazmat\backends\openssl\ciphers.pyo", line 137, in finalize
    
cryptography.exceptions.InvalidTag: 

Unhandled Error
Traceback (most recent call last):
  File "threading.pyo", line 763, in run
    
  File "Tribler\Core\Utilities\twisted_thread.pyo", line 55, in _reactor_runner
    
  File "twisted\internet\base.pyo", line 1192, in run
    
  File "twisted\internet\base.pyo", line 1201, in mainLoop
    
--- <exception caught here> ---
  File "twisted\internet\base.pyo", line 797, in runUntilCurrent
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1064, in on_data
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 722, in relay_packet
    
  File "Tribler\community\tunnel\tunnel_community.pyo", line 1255, in crypto_relay
    
  File "Tribler\community\tunnel\crypto\tunnelcrypto.pyo", line 81, in decrypt_str
    
  File "cryptography\hazmat\primitives\ciphers\base.pyo", line 174, in finalize
    
  File "cryptography\hazmat\backends\openssl\ciphers.pyo", line 137, in finalize
    
cryptography.exceptions.InvalidTag: 
ERROR   1448953861.10         handler:455   TFTP[36497 C 41.35.15.61:1036][15ae07f1d927be1b153e73dbdea6cc8ce6ae68fb.torrent] file size 13181 doesn't match expectation 16253

**Comment by whirm** [comment]
I'm guessing this is for version 6.4.3? The pre-release version should contain fixes for this.

**Comment by colin1497** [comment]
No, but it’s an installation that was previously updated from 6.4.3, as I’ve reverted back to that installation after creating several clean installs for testing purposes.

**Comment by whirm** [comment]
One of the errors you are getting is on another issue, the other should be fixed by #1793, but it may be covering another bug. So if you get different stack traces once this is merged, please, create a new issue with it.

Thanks!

**Comment by whirm** [comment]
@colin1497 Could you test this build? If it still fails, the logs should provide some useful info to fix one of the errors.

https://jenkins.tribler.org/job/Publish_tribler_next/162/

**Comment by colin1497** [comment]
I can try it, but I'm not sure I have a rock solid way to reproduce the above logs.  I haven't had a log file since those posted, but I haven't done much lately...

**Comment by colin1497** [comment]
I can try it, but I'm not sure I have a rock solid way to reproduce the above logs.  I haven't had a log file since those posted, but I haven't done much lately...

**Comment by whirm** [comment]
@colin1497 OK, just keep using that build (or any other more recent from that branch) if it's all the same to you in case you trigger it again.

Thanks!

**Comment by whirm** [comment]
I'm closing this one as it's a mix of different stuff and some of them are fixed.

If you get any stacktraces with the latest pre-release build, please create new issues for them separately.

Thanks!
